content/docs/scripts/polyfilling.md

```diff
@@ -0,0 +1,28 @@
+---
+title: 'Polyfilling'
+order: 9
+---
+
+In the past the `@babel/polyfills` was used, but it became obsolete. It is necessary to manually import the polyfills if needed.
+For this case we have created package `react-union-polyfills`.
+
+##Usage
+
+To use stable polyfills, just import `importPolyfills` function from `react-union-polyfills` in root package
+
+```js
+import { importPolyfills } from 'react-union-polyfills';
```

Comment:
you don't need to run them?

```
importPolyfills()
```

Reply:
No, because function for import stable polyfills is automatically called, when package is imported